app/api/router.py

```
@@ -0,0 +1,101 @@
+    ConversationStatus,
+)
+
+router = APIRouter()
+
+class SimpleConversationRequest(BaseModel):
+    project_ids: List[str]
+    agent_ids: List[str]
+
+async def get_api_key_user(
+    x_api_key: Optional[str] = Header(None),
+    db: Session = Depends(get_db)
```

Ruff (0.8.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)